tests/components/manual/test_alarm_control_panel.py

```diff
@@ -1461,8 +1462,8 @@ async def test_arm_away_after_disabled_disarmed(hass):
 
         state = hass.states.get(entity_id)
         assert STATE_ALARM_PENDING == state.state
-        assert STATE_ALARM_ARMED_AWAY == state.attributes["pre_pending_state"]
-        assert STATE_ALARM_TRIGGERED == state.attributes["post_pending_state"]
+        assert STATE_ALARM_ARMED_AWAY == state.attributes["previous_state"]
+        assert STATE_ALARM_TRIGGERED == state.attributes["next_state"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5411564)